src/libPersistence/BlockStorage.cpp

```
@@ -1648,6 +1649,24 @@ bool BlockStorage::GetMinerInfoShards(const uint64_t& dsBlockNum,
   return found;
 }
 
+bool BlockStorage::PutContractCreator(const dev::h160 address, const dev::h256 txnHash) {
+  if (!m_contractCreatorDB) {
+    return true;
```

shouldn't this be false?

---

So `m_contractCreatorDB` won't be initialised when `!LOOKUP_NODE_MODE`. But I decided to still call `PutContractCreator` unconditionally and just silently return if we weren't a lookup node. I can change the setup if you think its unclear?